DbgShell/DbgShell.csproj

```diff
@@ -129,13 +130,13 @@
   <PropertyGroup>
     <FileDescription>Main DbgShell EXE: implements the PowerShell host.</FileDescription>
   </PropertyGroup>
-  <Import Project="..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets" Condition="Exists('..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets')" />
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets'))" />
+    <Error Condition="!Exists('..\packages\Nerdbank.GitVersioning.2.3.125\build\Nerdbank.GitVersioning.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Nerdbank.GitVersioning.2.3.125\build\Nerdbank.GitVersioning.targets'))" />
```

Comment:
>Nerdbank [](start = 43, length = 8)

Let's revert the Nerdbank.GitVersioning changes for now. I would like to move to a more standard version-stamping solution (such as Nerdbank.GitVersioning), but a) I'd like it to be in a separate change, b) I'd like to move all components at the same time, and c) I don't know if NB.GV is ready yet (last I checked, it didn't handle C++/CLI binaries, but that was maybe a few years ago).

The AddGitVersionInfo stuff is weird and crusty (I know because I wrote it), but for now it produces results that I like. For instance, before you revert this change, run `Get-DbgShellVersionInfo` in a debug build and compare the info given for DbgShell.exe versus DbgProvider.dll.

Reply:
🤬 how the hell did I manage to get that committed to this branch.... or any branch. I'd just tried that offhand to see what the difference was, I thought I reverted it shortly aftward 😩